_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: Using Prometheus to visualize metrics
+nav_order: 40
+redirect_from:
+  - /observing-your-data/prometheusmetrics/
+---
+
+# Using Prometheus to visualize metrics
+Introduced 2.4
+{: .label .label-purple }
+
+Starting with OpenSearch 2.4, you can visualize metrics data from the open-source [Prometheus](https://prometheus.io/) monitoring solution as well as from log data aggregated within OpenSearch, unlocking new observability capabilities that allow you to analyze and correlate data across logs and traces as well as metrics. Previously, the OpenSearch observability plugin only allowed you to ingest and visualize logs and traces from your monitored environments. With this feature, you can observe your digital assets with more granularity, gain deeper insight into the health of your infrastructure, and better inform root cause analysis.
+
+![Prometheus and Metrics]({{site.url}}{{site.baseurl}}/images/metrics/metricsgif.gif)
+
+## Creating a connection to Prometheus
+
+You can create a connection from [Prometheus](https://prometheus.io/) to OpenSearch using the SQL plugin in order to view metrics collected from Prometheus in OpenSearch Dashboards.
```

Comment:
"in order to" can be boiled down to "to". Even better, replace with "and". Does that read all right?

"You can create a connection from Prometheus to OpenSearch using the SQL plugin and view metrics collected from Prometheus in OpenSearch Dashboards."